holoviews/operation/datashader.py

```diff
@@ -332,3 +332,70 @@ def _process(self, element, key=None):
         agg = aggregate._process(self, element, key)
         shaded = shade._process(self, agg, key)
         return shaded
+
+
+
+class dynspread(ElementOperation):
+    """
+    Spreading expands each pixel in an Image based Element a certain
+    number of pixels on all sides according to a given shape, merging
+    pixels using a specified compositing operator. This can be useful
+    to make sparse plots more visible. Dynamic spreading determines
+    how many pixels to spread based on a density heuristic.
+    """
+
+    how = param.ObjectSelector(default='source',
+                               objects=['source', 'over',
+                                        'saturate', 'add'],
+                               doc="""
+        The name of the compositing operator to use when 
+        combining pixels.       
+    """)
+
+    max_px = param.Integer(default=2, doc="""
+        Maximum number of pixels to spread on all sides.
+    """)
+
```

Comment:
Is this the HoloViews docstring style?  It's up to you, but personally I much prefer to have it tighter:

```
max_px = param.Integer(default=2, doc="""
         Maximum number of pixels to spread on all sides.""")
```